python/pyspark/sql/catalog.py

```diff
@@ -133,6 +133,35 @@ def listColumns(self, tableName, dbName=None):
                 isBucket=jcolumn.isBucket()))
         return columns
 
+    def tableExists(self, tableName, dbName=None):
+        """Check if the table or view with the specified name exists.
+        This can either be a temporary view or a table/view.
+
+        .. versionadded:: 3.3.0
+
+        Parameters
+        ----------
+        tableName : string
```

Comment:
```suggestion
        tableName : str
```

Reply:
Updated, as well as the documentation for dbName